samples/net/sockets/sntp_client/prj.conf

```diff
@@ -0,0 +1,36 @@
+# General config
+CONFIG_NEWLIB_LIBC=y
+
+# Networking config
+CONFIG_NETWORKING=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_IPV6=y
+CONFIG_NET_UDP=y
+CONFIG_NET_SHELL=y
+
+# Sockets
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_POSIX_NAMES=y
+CONFIG_NET_SOCKETS_POLL_MAX=4
+
+# Network driver config
+CONFIG_TEST_RANDOM_GENERATOR=y
+
+# Network address config
+CONFIG_NET_CONFIG_SETTINGS=y
+CONFIG_NET_CONFIG_NEED_IPV4=y
+CONFIG_NET_CONFIG_NEED_IPV6=y
+CONFIG_NET_CONFIG_MY_IPV4_ADDR="192.0.2.1"
+CONFIG_NET_CONFIG_PEER_IPV4_ADDR="192.0.2.2"
+CONFIG_NET_CONFIG_MY_IPV6_ADDR="2001:db8::1"
+CONFIG_NET_CONFIG_PEER_IPV6_ADDR="2001:db8::2"
+
+# SNTP
+CONFIG_SNTP=y
+
+CONFIG_MAIN_STACK_SIZE=2048
+# Network debug config
+CONFIG_NET_LOG=y
+CONFIG_LOG=y
+CONFIG_NET_SOCKETS_LOG_LEVEL_DBG=n
```

Comment:
I have changed something like that today morning.

Reply:
Any issues with that?

Reply:
For me it implies that one must disable `NET_SOCKETS_LOG_LEVEL_DBG` for this sample to work, which seems wrong. I personally would still strip this line away but if the majority wants to leave something in then maybe use the commented out version, as used before.

Reply:
> For me it implies that one must disable `NET_SOCKETS_LOG_LEVEL_DBG` for this sample to work,

really??

Reply:
Sorry, if that sounded dumb. :confused: It was just my initial thought (especially from a new users point of view).

I think @nashif has a very good explanation, see his comment above.